### PR TITLE
Update feature-summary.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor formatting fixes.

### CODE REVIEW

File: docs/feature-summary.md [hunk 1]
```diff
@@ -20,31 +18,26 @@ Each user in Mattermost is connected with their own personal Jira account and no
 
 ### Create Jira issues
 
-Create Jira issues from scratch or based off of a Mattermost message easily
-
-Without leaving Mattermost's UI, quickly select the project, issue type and enter other fields to create the issue.
+Create Jira issues from scratch or based off a Mattermost message easily. Without leaving Mattermost's UI, quickly select the project, issue type, and enter other fields to create the issue.
```
Comment (jfrerich) [inline docs/feature-summary.md:21]:
```suggestion
Create Jira issues from scratch or based off a Mattermost message. Without leaving Mattermost's UI, quickly select the project, issue type, and enter other fields to create the issue.
```

**Comment by justinegeffen** [comment]
> LGTM! Approving, but looks like ci failed. Let me know if you need help resolving!

Thank you! I'm not sure why it's failing, so any tips would be appreciated! :)

**Comment by hanzei** [comment]
@justinegeffen CI is green now

**Pull request merged**